lib/display.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'grid'
+require_relative 'converter'
+
+class Display
+
+  def initialize(output = $stdout)
+    @output = output
+  end
+
+  def display_grid(number)
```

Comment:
I think that this method should receive a grid and a converter as arguments rather than creating new instances inside it.